### DISCUSSION TITLE
Enshrouded

### DISCUSSION THREAD

**Discussion by PR3SIDENT** [gameserverapp/Platform#625] [Game request]
https://www.youtube.com/watch?v=dUj1Kh80GKg&ab_channel=Enshrouded
https://store.steampowered.com/app/1203620/Enshrouded/

This game is going to be HELLA DOPE in the Survival Crafting genre, we played a ton of the demo and its mega, mega, mega good.

**Comment by LimitlessDrag0n** [comment]
Agreed. The Demo was huge and has potential! I hope they have dedi support then

**Comment by ad3d0** [comment]
It's starting in EA today, dedicated files will be available right after launch.
Here are some useful info on server setup, as I see even with some docker container examples there.

**Reply by mvaessen** [reply]
Unfortunately, all examples use WINE.

**Comment by mvaessen** [comment]
This game is now available on the dashboard, for Windows machines only.

Please note that this game is __very__ early access:
- No rcon
- No manual world saves (so no reliable backups)
